src/plugins/irc-events/link.js

```diff
@@ -106,6 +106,23 @@ function parse(msg, preview, res, client) {
 
 		break;
 
+	case "audio/midi":
+	case "audio/mpeg":
+	case "audio/mpeg3":
+	case "audio/ogg":
+	case "audio/wav":
+	case "audio/x-mid":
+	case "audio/x-midi":
+	case "audio/x-mpeg":
+	case "audio/x-mpeg-3":
+		if (res.size > (Helper.config.prefetchMaxImageSize * 4028)) {
```

Comment:
How do browsers deal with preloading audio data? Does it download the entire audio file as soon as it sees the `audio` tag or downloads just enough to display the duration?

Reply:
Browsers should default to fetching metadata, but I can add a ‘preload’ attribute to specify

Reply:
My point is, if browsers just fetch the metadata, don't really need to limit the size.

You do need to abort the request after content-type was fetched by the way.

Reply:
MDN says browsers should should default to metadata, and says that the preload attribute acts as a hint to the browser — should we trust the browser?

Reply:
MDN mentions:

> If not set, its default value is browser-defined (i.e. each browser may have its own default value). The spec advises it to be set to `metadata`.

I think it's fair to have `preload="metadata"`.

Reply:
What if we went with `preload=none` to avoid unnecessary requests? When I tried this PR Chrome didn't display duration half the time anyway.

Reply:
Time won’t be loaded then, and I think that’s pretty important

Reply:
added `preload="metadata"`